CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Support for OpenShift 4.8 has been certified as of this release.
```

Comment:
Has it actually been "certified"? Are these changes enough to declare this? JW

Reply:
I may have jumped the gun here, but I was under the impression that this was already certified for 4.8, but cleaning up the tests was just a matter of ensuring we tested against 4.8. We may want to have someone clarify this.

Reply:
Yes please do - if we just started testing on 4.8 now then I'm not sure how it could have been compatibility "certified" before :)